### DISCUSSION TITLE
Unable to apply in Octernship!!

### DISCUSSION THREAD

**Discussion by Rookie0208** [community/community#57937] [GitHub Education]
### Hi everyone,

I'm new to this program and i want to apply for the octernship but they are showing that i do not meet the eligibility criteria. 
I've a verified account on github according to their criteria. 
Please suggest how can i apply for this.

**Comment by tuzun89** [comment]
You need to be enrolled in uni from the list of countries currently participating in the program.

**Reply by SoufiyaneOuali** [reply]
If you are encountering an issue with meeting the eligibility criteria for an internship program, despite having a verified GitHub account, there are a few steps you can take:

Review the eligibility criteria: Carefully read through the requirements and ensure that you meet all the necessary qualifications and prerequisites. Check if there are any specific educational, experience, or skill requirements that you may have missed.

Contact the program organizers: Reach out to the program organizers or the contact provided on the internship application page. Explain your situation and inquire about the specific reason for not meeting the eligibility criteria. They may be able to provide you with more information or clarify any misunderstandings.

Showcase your skills and experiences: If you believe you have relevant skills and experiences that align with the internship program, prepare a compelling application highlighting your achievements, projects, and contributions. Emphasize how these experiences make you a suitable candidate despite not meeting one specific criterion.

Seek alternative opportunities: If you are unable to apply for the specific internship program, consider exploring other internship opportunities in your field of interest. There are often multiple organizations offering similar programs, and expanding your search can increase your chances of finding a suitable position.

Remember, each internship program has its unique criteria and selection process, so it's essential to thoroughly understand the requirements and communicate effectively with the program organizers to address any concerns or issues. Good luck with your internship application!
